lib/utils/flattenArray.js

```diff
@@ -1,16 +0,0 @@
-'use strict';
-
-/**
- * Convert the specified value to an array. If an array is specified, the array is returned as-is.
```

Comment:
Looking at the description and the implementation, it seems that, despite the name, it's different from `Array.prototype.flat`.